sql/schema.sql

```diff
@@ -549,6 +549,15 @@ create table pranger (
 create index pranger_at_idx
           on pranger (at desc);
 
+-- Alliance members
+
+create table user_password_reset_tokens (
+  user_id bigint references users on delete cascade unique not null,
+  encrypted_token text not null,
+  expire_at timestamptz not null,
+  primary key (user_id, encrypted_token)
```

Comment:
The corresponding entity says that the primary key is only the user, but here `encrypted_token` is used as well. Since for each user there is only one token, we probably could use only the `user_id` as primary key.